packages/block-library/src/quote/edit.js

```diff
@@ -48,6 +48,7 @@ export default function QuoteEdit( { attributes, setAttributes, isSelected, merg
 								citation: nextCitation,
 							} )
 						}
+						noFocusOnMount
```

Comment:
Will this not have an impact on the web side?

Reply:
What do you have in mind Sérgio?

Reply:
Just fearful that adding a prop could have some kind of impact. But if it's ok for @gziolo it's ok for me :).

Reply:
It's easy to add new props, but it's nearly impossible to remove them later in the WordPress world. Backward compatibility is very important for the project. That's why we always seek to find the most minimal public API we can think of. If there is no other way of handling that with mobile implementation, can we at least use `__unstableMobileNoFocusMount` name to indicate that this might change in the future?

See:  https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#experimental-and-unstable-apis

> An **unstable API** is one which serves as a means to an end. It is not desired to ever be converted into a public API.

Reply:
Aha, that makes sense @gziolo , I'll go with `__unstableMobileNoFocusMount` so to move this PR forward (it's an important fix for mobile actually) and that will allow us to more easily revise the implementation. Will do that change now.

Reply:
Done with 2ac9adfa9.

Reply:
Nice, thanks!